packages/ui/cart/CartTable.tsx

```diff
@@ -43,6 +43,16 @@ const CartTable = () => {
   };
 
   const web = useMediaQuery(`(min-width: ${DEFAULT_THEME.breakpoints.sm})`);
+  const totalItems = cartItems.reduce(
+    (total, item) => total + item.quantity,
+    0,
+  );
+  const subtotal = calculateSubtotal(cartItems);
```

Comment:
please move computation to back-end

Reply:
computation will be move to back-end in api implementation. This will be another ticket